hedera-node/hedera-app/src/test/java/com/hedera/node/app/workflows/handle/record/SingleTransactionRecordBuilderTest.java

```diff
@@ -58,6 +57,7 @@
 @ExtendWith(MockitoExtension.class)
 public class SingleTransactionRecordBuilderTest {
     public static final Instant CONSENSUS_TIME = Instant.now();
+    public static final Instant PARENT_CONSENSUS_TIME = Instant.now();
```

Comment:
We need to ensure that `CONSENSUS_TIME` and `PARENT_CONSENSUS_TIME` are different. Something like CONSENSUS_TIME.plusNanos(1L) would guarantee that.

Reply:
Updated.